### ISSUE TITLE
Accessibility - VoiceOver on Apple OS

### ISSUE THREAD

**Issue by adn-aceres** [microsoft/BotFramework-WebChat#2318]
🚨 The issue tracker is not for implementation questions 🚨

[Question]
**Accessibility / VoiceOver Mac OS / Last new answer (read)**

We also were thinking about Web Accessibility. We have tested it with VoiceOver and as we see it is not accessible especially for the people who cannot see. It would be great when the chatbot can jump to the next answer that the VoiceOver also can recognize / identify the newest answer from the chatbot and read it. I still don't know what the solution would be but anyway it is always good to share about it together.

Thank you for your answer in advance.

Kind regards,
André

**Comment by cwhitten** [comment]
Hello @adn-aceres. BF WebChat is regularly audited to meet Accessibility compliance requirements. Our last audit was in March 2019 and we received a WCAG 2.0 AA rating.

That said, we will do some testing with Voiceover to see if there is something causing what you are seeing and report back.

**Comment by corinagum** [comment]
Hi @adn-aceres, sorry for the delay in getting back to you. I did a test of basic functionality of Web Chat, including the accessibility changes that have been recently made, and I did not encounter abnormalities. If you have more specific details on what is not accessible, please feel free to file a new issue. 

Thank you!
